CI/steam/scripts/uninstall.bat

```diff
@@ -0,0 +1,33 @@
+@echo off
+@cd /d "%~dp0"
+goto checkAdmin
+
+:checkAdmin
+	net session >nul 2>&1
+	if %errorLevel% == 0 (
+		echo.
+	) else (
+		echo Administrative rights are required, please re-run this script as Administrator.
```

Comment:
"Administrative rights are required. Please re-run this script as Administrator."